helm/node-exporter-app/templates/daemonset.yaml

```diff
@@ -70,11 +68,16 @@ spec:
         - '--collector.xfs'
 
         - '--no-collector.btrfs'
+        - '--no-collector.bonding'
         {{- if ne (include "provider" .) "azure" }}
         - '--no-collector.diskstats'
         {{- end }}
+        - '--no-collector.hwmon'
         - '--no-collector.infiniband'
         - '--no-collector.ipvs'
+        - '--no-collector.mdadm'
+        - '--no-collector.nfs'
+        - '--no-collector.nfsd'
```

Comment:
I wonder if we should keep these enabled on KVM like Marcel said (following the azure-specific collector above). Could you see if the alert is firing on KVM installations for these two collectors?

Reply:
Possibly `bonding` as well. This might depend on whether we're running in a VM or on bare metal.

Reply:
Same with `hwmon`. I didn't realise that there is such a big difference between kvm installations. There is even a difference between master and worker nodes on the same installation - the file for `nfs` for example is only on the masters. I'll go through the alerts and rework the list.